pkg/driver/driver.go

```diff
@@ -146,9 +146,17 @@ func WithEndpoint(endpoint string) func(*DriverOptions) {
 	}
 }
 
+func WithExtraTags(extraTags map[string]string) func(*DriverOptions) {
+	return func(o *DriverOptions) {
+		o.extraTags = extraTags
+	}
+}
+
 func WithExtraVolumeTags(extraVolumeTags map[string]string) func(*DriverOptions) {
 	return func(o *DriverOptions) {
-		o.extraVolumeTags = extraVolumeTags
+		if o.extraTags == nil {
+			o.extraTags = extraVolumeTags
```

Comment:
nit: could you add a warning saying this flag is deprecated, that it will be removed in the future and that the the user should use `--extra-tags` instead?